CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 * [#174](https://github.com/alexa-js/alexa-app/pull/174): Always enfore strict header checking - [@tejashah88](https://github.com/tejashah88).
 * Your contribution here.
 
+### 3.1.3 (February 16, 2017)
```

Comment:
This should really happen when you do an actual release, but I'll let it slide ;)